.github/workflows/backup-database.yml

```
@@ -0,0 +1,57 @@
+---
+name: Perform Django dbbackup and mediabackup
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: string
+        default: dev
+
+jobs:
+  backup-database:
+    name: Backup ${{ inputs.environment }} database
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      space: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Bind backups bucket to ${{ inputs.environment }} application
+        if: ${{ inputs.environment == 'dev' || inputs.environment == 'production' }}
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: gsa-tts-oros-fac
+          cf_space: ${{ env.space }}
+          command: cf bind-service gsa-fac backups -w
+
+      - name: Backup the database in ${{ inputs.environment }}
+        if: ${{ inputs.environment == 'dev' || inputs.environment == 'production' }}
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: gsa-tts-oros-fac
+          cf_space: ${{ env.space }}
+          command: cf run-task gsa-fac -k 2G -m 2G --name pg_backup --command "./backup_database.sh ${{ env.space }}"
+
+      - name: Sleep for 5 minutes to ensure backup occurs
+        if: ${{ inputs.environment == 'dev' || inputs.environment == 'production' }}
+        run: sleep 5m
+        shell: bash
```

```suggestion
      - name: Sleep for 5 minutes to ensure backup occurs
        if: ${{ inputs.environment == 'dev' || inputs.environment == 'production' }}
        run: sleep 5m
        shell: bash
```

This does need to be removed at some point, but for whatever reason, I am unable to use `cf auth` and then `cf curl /v3/apps/$(cf app gsa-fac --guid)/task` with my cf account. Doing so will return a json we can parse that retrieves the status of the task, and then that will loop until `status == 'SUCCEEDED'`

Taking this as an enhancement down the line